src/main/java/seedu/address/model/person/Note.java

```diff
@@ -36,7 +36,7 @@ public boolean isMatch(Object otherValue) {
 
         String other = (String) otherValue;
 
-        return this.getValue().toLowerCase().contains(other.toLowerCase());
+        return this.getValue().trim().toLowerCase().contains(other.trim().toLowerCase());
```

Comment:
The `parseNote` method in `ParserUtil.java` already contains the logic to trim the note before instantiating a `Note` instance. Seems redundant to add it here again.  However, `NoteCommandParser.java` does not appear to be using the `parseNote` method and instead creates a new note without trimming.